src/ProjectTemplates/test/Templates.Tests/GrpcTemplateTest.cs

```diff
@@ -35,7 +35,7 @@ public ITestOutputHelper Output
     }
 
     [ConditionalTheory]
-    [SkipOnHelix("Not supported queues", Queues = "All.OSX;" + HelixConstants.Windows10Arm64 + HelixConstants.DebianArm64)]
+    [SkipOnHelix("Not supported queues", Queues = "windows.11.arm64.open;" + HelixConstants.Windows10Arm64 + HelixConstants.DebianArm64)]
```

Comment:
Can we add this value to HelixConstants?

Reply:
Probably. But first, I'd like to ensure this change fixes the problem. The problem is that the Helix queue only runs on full CI (i.e. not PR CI). Need to merge to check.

Reply:
There are ways to run it without merging PRs, oh well

Reply:
Ok!

https://github.com/dotnet/aspnetcore/issues/45885 😄